config/collector.yaml

```diff
@@ -192,4 +192,4 @@ service:
           exporter:
             otlp:
               protocol: grpc/protobuf
-              endpoint: ${env:MY_POD_IP}:14317
+              endpoint: ${env:MY_POD_IP}:14317
```

Comment:
Please add newline at the end of the file.

This file does not seem to contain any other changes related to version bump, so it should be untouched.